Makefile

```diff
@@ -40,7 +40,7 @@ lint: validate-go-version
 build: validate-go-version clean $(BINARY)
 
 $(BINARY):
-	CGO_ENABLED=0 $(GO) build -a -installsuffix cgo -ldflags="-X main.VERSION=${VERSION}" -o $@ github.com/oauth2-proxy/oauth2-proxy/v7
+	CGO_ENABLED=0 $(GO) build -a -installsuffix cgo -ldflags="-X github.com/oauth2-proxy/oauth2-proxy/v7/pkg.version.VERSION=${VERSION}" -o $@ github.com/oauth2-proxy/oauth2-proxy/v7
```

Comment:
github.com/oauth2-proxy/oauth2-proxy/v7/pkg.version.VERSION

Is the . between pkg and version correct here?

In the import statements you use "github.com/oauth2-proxy/oauth2-proxy/v7/pkg/version"

Reply:
I'll test it on my end tomorrow